docs/Repository/README.md

```diff
@@ -0,0 +1,46 @@
+## Garden Linux Repositories
+
+Garden Linux is a Debian-based Operating System. Therefore it uses apt.
+
+The location of sources are at `/etc/apt/sources.list` respectively in the folder `/etc/apt/sources.list.d/`
+
+#### Default Repo:
+	deb http://repo.gardenlinux.io/gardenlinux dev main
+
+#### Generally:
+
+	deb http://repo.gardenlinux.io/gardenlinux $GARDENLINUX_EPOCH main
+
+`$GARDENLINUX_EPOCH` is generally `dev`, but the Garden Linux Version number can be defined here. It is the number of days since first Garden Linux release.
+
+<details>
+  <summary>More Info</summary>
+  
+Try running [bin/garden-version](bin/garden-version), to get the $GARDENLINUX_EPOCH value:
+  ```
+# ./bin/garden-version
+    dev
+
+# ./bin/garden-version --major
+    730
+
+# ./bin/garden-version --minor
+    0
+  ```
+- --major prints the number of days after release and is the main version number of Garden Linux.
+- --minor is mainly for security updates for older major versions.
+
+The Source Line for this example should look like:
+
+    deb http://repo.gardenlinux.io/gardenlinux 730.0 main
+	
+
+</details>
+
+## Misc
+
+- currently we keep old packages in `dev` dist, and I think we are keeping it that way for now.
```

Comment:
If the package name changes, we do not delete old package names. This is currently (afaik) only the case for our kernel related packages, that contain the full version string


>  "I think we are keeping it that way for now." 

In a documentation I would avoid to tell from a "me" perspective. 

The fact that we are keeping it for now, does not need to be explicitly written out. The documentation will be updated once we change this.

Reply:
Good point, thank you. I will keep that in mind for future documentation!